modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -1,101 +1,127 @@
-[eq1] can be written as:
+is a solution to [eq1] where $B$ is a complex number depending on the boundary conditions, $k^2=\omega^2\rho/E$ is the wave number, $\omega=2 \pi f$ where $f$ is the frequency, and $i=\sqrt{-1}$.  
+By assuming [eq2] is a a solution to [eq1] we can solve [eq1] in the frequency domain by taking a Fourier transform of $u(x,t)$ to get $U(x,\omega)$.  The frequency domain version of [eq1] is the Helmholtz equation given by
 \begin{equation}
-     -\nabla \cdot \sigma + \frac{\partial^2 u}{\partial t^2} = 0 \label{eq3}
+    E\frac{\partial^2 U}{\partial x^2}-\rho\omega^2 U=0.
+\label{eq3}
 \end{equation}
+[eq3] is easily solved in MOOSE where $U$ is the state variable.  The first term on the right hand side is still captured by the [StressDivergence.md] kernel.  The second RHS term, $\rho\omega^2 U$, is captured by the [Reaction](/Reaction.md) kernel where the `Reaction` rate is given by $-\omega^2$.  [eq3] is only valid for small displacements and linear elasticity.  A damping term can also be included in [eq1] and [eq3].
 
-Now, we consider the plane wave $u= r e^{i(\omega t - k x)}$ is a solution for [eq3], with $r$ being the complex amplitudes, and $k$ is the wave number and $i=\sqrt{-1}$. As we noted before Young's modulus and the density are constants and equal to unity, thus, $c_p=1$ where $c_p=\sqrt \frac{E}{\rho}$ is the longitudinal wave velocity .
-
-!alert note
-One can verify that $r e^{i(\omega t - k x)}$ is a solution for [eq3]$\Longleftrightarrow$ $k^2= \frac{\omega^2}{c_p^2}$ by direct substitution.
-
-Now, [eq2] and the associated BCs. and can be written as,
+The boundary conditions also need to be converted to the frequency domain through a Fourier transform.  A time harmonic Neumann BC given by
 \begin{equation}
-     -\nabla \cdot \sigma- \omega^2 u = 0 \label{eq4}
+    f= A \, cos(2 \pi f t) \qquad \text{at} \, x=L
+    \label{eq4}
 \end{equation}
-
+with amplitude $A$ and frequency $f$, is transformed to
 \begin{equation}
-    \nabla u= ik \, u \, \qquad \text{at} \, x=0,
-    \\
-    u= A \, cos(2 \pi f t) \, \qquad \text{at} \, x=L,\label{eq5}
+    F= A \, \qquad \text{at} \, x=L
+    \label{eq5}
 \end{equation}
 
-Transforming [eq4] and the associated boundary conditions in [eq5] in time to the frequency domain, we obtain:
-\begin{equation}
-     -\nabla \cdot \hat \sigma- \omega^2  U = 0 \label{eq6}
-\end{equation}
-with the following boundary conditions:
+# Cantilever Beam Example
+
+The cantilever beam shown in [cantilever] is subjected to a time harmonic force on the right side in the out-of-plane and vertical directions.  In this example the frequency of the time varying load is swept over a range.  The displacement at the midpoint of the beam is recorded at each frequency.  This type of output, displacement as a function of frequency, is a frequency response function (FRF) or transfer function.  Displacement peaks in the FRF indicate natural frequencies.  
```

Comment:
"in the out-of-plane and vertical directions" are you applying two forces (one in the out of plane, and the other vertical (in-plane)?

Reply:
yes.  I am applying loads in the two directions tangent to the beam's axis.  I am not applying a load along the axis of the beam.

Reply:
Is there a reason you applied two loads perpendicular to each other? why not just vertical?

Also, the way I understood it when I read "out of plane" I understood it as out-of-plane with respect to the beam's cross-section. I think it may be more precise if we just add "tangential to the beam's cross-section" after out-of-plane.